client/analysisui/iconsupport.css

```diff
@@ -0,0 +1,40 @@
+/*textbox sizing*/
+.jmv-option-icon[data-icon=ContinuousVariable] {
```

Comment:
there's like this unwritten rule that we avoid capitals in html and css. you could make this consistent with existing stuff:

https://github.com/jamovi/jamovi/blob/master/client/main/tableview.css#L220